packages/workbox-core/test/bundle/node/controllers/test-WorkboxCore.js

```diff
@@ -15,6 +15,12 @@ describe(`WorkboxCore - ${process.env.NODE_ENV}`, function() {
       expect(core.INTERNAL).to.exist;
     });
 
+    it(`should return the same INTERNAL instance after first call`, function() {
+      const core = new WorkboxCore();
+      const internal = core.INTERNAL;
+      expect(core.INTERNAL).to.equal(internal);
```

Comment:
Not sure if it matters in this context, but my default is to use [`eql`](http://chaijs.com/api/bdd/#method_eql) (which tests for deep-equality) instead of `equal` whenever possible.